CHANGELOG.md

```diff
@@ -18,12 +18,16 @@ Changelog for NeoFS Node
 - Use iterators for container listings (#3196)
 - Also reloading shard mode with SIGHUP (#3192)
 - IR logs INFO its state on start based on read FS chain's internals (#3197)
+- Use list instead of maps for options in node config (#3204)
 
 ### Removed
 
 ### Updated
 
 ### Updating from v0.45.0
+The node config has been changed. Now instead of the fields `node.attribute_*` and 
+`storage.shard.*` there will be lists `node.attributes` and `storage.shard.shards`.
```

Comment:
`shard.shards` looks strange, but i understand this is cuz of existing `shard.default`. It can be one of:
1. `shard.list`
2. `shards_default` and `shards`

2 is more preferable to me. 1 is easier todo